src/AltinnCore/AltinnCore.Runtime.Db/Models/ReporteeElement.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AltinnCore.Runtime.Db.Models
+{
+    public class ReporteeElement
+    {
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
+
+        [JsonProperty(PropertyName = "reporteeId")]
+        public string ReporteeId { get; set; }
+
+        [JsonProperty(PropertyName = "serviceId")]
+        public string ServiceId { get; set; }
+
+        [JsonProperty(PropertyName = "serviceEdition")]
+        public string ServiceEdition { get; set; }
+
+        [JsonProperty(PropertyName = "reporteeElementId")]
+        public string ReporteeElementId { get; set; }
+
+        [JsonProperty(PropertyName = "reporteeElementType")]
+        public string ReporteeElementType { get; set; }
+
+        [JsonProperty(PropertyName = "reporteeElementStatus")]
+        public string ReporteeElementStatus { get; set; }
+
+        [JsonProperty(PropertyName = "createdDateTime")]
+        public DateTime CreatedDateTime { get; set; }
+
+        [JsonProperty(PropertyName = "createdBy")]
+        public string CreatedBy { get; set; }
+
+        [JsonProperty(PropertyName = "lastChangedDateTime")]
+        public DateTime LastChangedDateTime { get; set; }
+
+        [JsonProperty(PropertyName = "lastChangedBy")]
+        public string LastChangedBy { get; set; }
+
+        [JsonProperty(PropertyName = "dueDateTime")]
+        public DateTime DueDateTime { get; set; }
+
+        [JsonProperty(PropertyName = "visibleDateTime")]
+        public DateTime VisibleDateTime { get; set; }
+
+        [JsonProperty(PropertyName = "title")]
+        public string Title { get; set; }
+
+        [JsonProperty(PropertyName = "serviceOwner")]
+        public string ServiceOwner { get; set; }
+
+        [JsonProperty(PropertyName = "externalSystemReference")]
+        public string ExternalSystemReference { get; set; }
+
+        [JsonProperty(PropertyName = "visibleDateTime")]
+        public string VisisbleDateTime { get; set; }
+
+        [JsonProperty(PropertyName = "workflowId")]
+        public string WorkflowId { get; set; }
+
+        [JsonProperty(PropertyName = "currentWorkflowStep")]
+        public string CurrentWorkflowStep { get; set; }
+
+        [JsonProperty(PropertyName = "isDeleted")]
+        public bool IsDeleted { get; set; }
+
+        [JsonProperty(PropertyName = "isArchived")]
+        public bool IsArchived { get; set; }
+
+        [JsonProperty(PropertyName = "numberOfForms")]
+        public int NumberOfForms { get; set; }
+
+        [JsonProperty(PropertyName = "formId")]
+        public string[] FormId { get; set; }
+
+        [JsonProperty(PropertyName = "reAttachment")]
+        public FormAttachment REAttachment { get; set; }
```

Comment:
What is this attribute? Should probably have a more descriptive name

Reply:
Fixed